src/rules.cpp

```diff
@@ -349,6 +350,14 @@ bool Condition::matchesWindowrole(const Client* client) const {
     return matches(role.value());
 }
 
+bool Condition::matchesFixedSize(const Client* client) const
+{
```

Comment:
Minor style nitpick: most other methods in this file have the opening curly bracket on the same line as the method definition

Reply:
Fixed!